File: al_learner/keys.py
import os

pg_host = os.getenv('PGHOST')
pg_dbname = os.getenv('PGDATABASE')
pg_user = os.getenv('PGUSER')
pg_password = os.getenv('PGPASSWORD')
pg_port = os.getenv('PGPORT')
redis_host = os.getenv('REDIS_HOST')
redis_port = os.getenv('REDIS_PORT')
batch_size = os.getenv('BATCH_SIZE')
min_label_count = os.getenv('MIN_LABEL_COUNT')

File: api/keys.js
module.exports = {
    node_port: process.env.NODE_PORT,
    redisHost: process.env.REDIS_HOST,
    redisPort: process.env.REDIS_PORT,
    pgUser: process.env.PGUSER,
    pgHost: process.env.PGHOST,
    pgDatabase: process.env.PGDATABASE,
    pgPassword: process.env.PGPASSWORD,
    pgPort: process.env.PGPORT,
    jwtSecret: process.env.JWT_SECRET,
    batchSize: process.env.BATCH_SIZE,
    uncertaintyThreshold: process.env.UNCERTAINTY_THRESHOLD,
    activeLearning: process.env.ACTIVE_LEARNING
  };